src/components/TodoList/TodoList.tsx

```diff
@@ -25,24 +38,33 @@ export const TodoList: React.FC = () => {
         </thead>
 
         <tbody>
-          <tr data-cy="todo">
-            <td className="is-vcentered">1</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">delectus aut autem</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
+          {filtredArrayOfTodos.map(todo => (
+            <tr data-cy="todo">
```

Comment:
```suggestion
            <tr data-cy="todo" key={...}>
```
Consider using a key for rendering lists